netdata.spec.in

```diff
@@ -69,12 +69,14 @@ happened, on your systems and applications.
 
 %install
 rm -rf $RPM_BUILD_ROOT
-%{__make} %{?_smp_mflags} install \
-	DESTDIR=$RPM_BUILD_ROOT
+%{__make} %{?_smp_mflags} DESTDIR=$RPM_BUILD_ROOT install
```

Comment:
put this on a single line and reordered to follow best practices (looking at other Redhat official RPM specs)